sponsorship.md

```diff
@@ -0,0 +1,46 @@
+# Pangeo Sponsorship Policy
+
+Now that Pangeo has a 501c3 fiscal sponsor, it can receive donations. Individuals or institutions who make donations to the project are **Project Sponsors**. The purpose of this document is to define the relationship between the Project and Project Sponsors.
+
+## Types of Sponsorship
+
+There are two ways to sponsor the Project:
+1. Direct financial contribution - a direct donation made via [NumFocus](https://numfocus.org/donate).
+2. In-kind contribution - goods or services provided to the project in support of its activities. Examples of in-kind contributions include:
+
+   - Personnel time dedicated to project support and administration. This excludes work on the project by Steering Council members.
+   - Computers, cloud credits, or other IT-related services paid for by the sponsor on behalf of the project.
+   - Access to facilities such as office space, meeting space, or shared conference booths.
+
+  When making an in-kind donation, the sponsor should notify NumFocus of the donation and quantify the value of the in-kind donation. (For donors in the United States, such donations are tax-deductible to the extent provided by law. As with any donation, donors should consult with their tax adviser about their particular tax situation.)
+
+The Steering Council or NumFOCUS may reject sponsorship offers that do not align with the Project values or mission.
+
+## Relationship between the Project and Sponsors
+
+The sponsorship relationship should be mutually beneficial to the Project and the Sponsor. The benefits of sponsorship to the Project are straightforward: more resources with which to achieve goals identified in the Project Roadmap and Project Mission. Some of the benefits to the Sponsor are:
+- Advancing the Project (which may be aligned with the sponsor’s goals and values)
+- Enhancing the Sponsor’s image via association with the Project
+- Reaching potential customers or clients via association with Project
+
+Sponsorship does not create any ability for the Sponsor to influence the Project governance or decision making, which remains solely with the Steering Council (with oversight from the NumFOCUS board for certain types of decisions).
+The project prefers unrestricted contributions, to be used at the discretion of the Steering Council to further the Project mission and roadmap.
+Restricted / conditional contributions, including in-kind donations and more complex partnership offers, will be reviewed and approved on a case-by-case basis by the Steering Council.
+To support work in a specific direction, grants are the preferred mechanism. 
+
+_Sponsorship does not imply any sort of endorsement by the Project of the Sponsor’s products or services._ While the Sponsor may publicly acknowledge its sponsorship, the sponsor may not use the Pangeo name or logo in its marketing materials in a way that suggests such endorsement.
```

Comment:
```suggestion
_Sponsorship does not imply any sort of endorsement by the Project of the Sponsor’s products or services._ While the Sponsor may publicly acknowledge its sponsorship, the Sponsor may not use the Pangeo name or logo in its marketing materials in a way that suggests such endorsement.
```

capitalize the Sponsor for consistency unless we decide to go with lowercase (and maybe plural) everywhere.